### ISSUE TITLE
logo display issue

### ISSUE THREAD

**Issue by alonewolfx2** [eadf/esp8266_pcd8544#2]
bitmap function not working correctly.it seems logo starting point wrong.(middle of screen)

**Comment by eadf** [comment]
Hm, there seems to be a missing PCD8544_gotoXY() inside PCD8544_lcdImage() 
PCD8544_lcdClear() is also missing that goto. (lcdClear is same as PCD8544_lcdImage but with just 0 data)
I'll check in a new version in a few minutes.

**Comment by alonewolfx2** [comment]
ok i am waiting new version.

**Comment by eadf** [comment]
ok, could you test it now? 

Btw. are you using the Makefile.linux? I've changed it so it only uses esptool.py and not "esptool-cs". I works for me, but it would be interesting to know if it works for you too

**Comment by alonewolfx2** [comment]
its working fine now. issue solved. 
i am using eclipse on windows, so i am using different makefile.
Edit: can you implement sleep functions for pcd8544?

**Comment by eadf** [comment]
Sure, if the device supports it it should be no problem. 
I'll look into that.

**Comment by eadf** [comment]
The device does not support any sleep functions, but i've added a PCD8544_setContrast() function that you can use to mimic sleep. `PCD8544_setContrast(0);`

The backlight is simply a led. The pcd8544 is not even aware that it exists.

**Comment by alonewolfx2** [comment]
thank you for this. what is range for contrast 0-?

**Comment by eadf** [comment]
0x00 - 0x7f. 
But the useful values are about 40-60. I've added setContrast to [the esp8266_pcd8544_example demo](https://github.com/eadf/esp8266_pcd8544_example) so you can see what effect different values has.

**Comment by alonewolfx2** [comment]
i didnt saw difference between 40 and 68 but 0 value working like sleep. maybe my module not working correctly.

**Comment by eadf** [comment]
Maybe there is a big difference between the lcds. I've only tested the one i got. 
I limited the range because i was just seeing a lot of black and nothing at all..

replace `uint8_t contrast = ((loopIterations << 2 ) & 0x1f) + 40;` at line 78 (user_main.c)
with `uint8_t contrast = (loopIterations << 2) & 0x7f;` to see the full range of contrast.

It would be interesting to see what the visible range of your lcd is.
